relay-server/src/endpoints/batch_metrics.rs

```diff
@@ -0,0 +1,29 @@
+use axum::http::StatusCode;
+use axum::response::IntoResponse;
+use relay_config::EmitOutcomes;
+use serde::{Deserialize, Serialize};
+
+use crate::actors::processor::ProcessBatchedMetrics;
+use crate::extractors::{SignedBytes, StartTime};
+use crate::service::ServiceState;
+
+#[derive(Debug, Serialize, Deserialize)]
+struct SendMetricsResponse {}
+
+pub async fn handle(
+    state: ServiceState,
+    start_time: StartTime,
+    body: SignedBytes,
+) -> impl IntoResponse {
+    if !body.relay.internal || state.config().emit_outcomes() != EmitOutcomes::AsOutcomes {
+        return StatusCode::FORBIDDEN.into_response();
+    }
+
+    state.processor().send(ProcessBatchedMetrics {
+        payload: body.body,
+        start_time: start_time.into_inner(),
+        sent_at: None,
```

Comment:
Any reason to not set `sent_at`?

Reply:
We don't have it in that endpoint. Since this endpoint is just for inter-Relay communciation, there's no clock drift correction necessary.

@jjbayer suggested to still implement clock drift in the internal message, so we don't forget about it once we use the message from some other place.